pom.xml

```diff
@@ -64,7 +64,7 @@
     <shadeBase>net.snowflake.ingest.internal</shadeBase>
     <slf4j.version>1.7.36</slf4j.version>
     <snappy.version>1.1.10.1</snappy.version>
-    <snowjdbc.version>3.13.30</snowjdbc.version>
+    <snowjdbc.version>3.13.33</snowjdbc.version>
```

Comment:
I don't see all the changes as described in the description, did i miss anything?

Reply:
I thought we weren't going to upgrade JDBC. Is upgrading JDBC required for fixing these vulns?

Reply:
We need 3.13.31 at a minimum to resolve the json-smart vulnerability, however there are a couple of vulnerable packages I've found in the latest 3.13.33 release